### ISSUE TITLE
Doorbot 4 not working

### ISSUE THREAD

**Issue by JackiePease** [DoESLiverpool/somebody-should#1420]
Doorbot 4 (inside the front shutter) isn't working. 
Some people who are coming in to work on the COVID 19 project aren't aware that you can still get in by using Doorbot 1 again

**Comment by johnmckerrell** [comment]
What happened with this? Is it working now?

**Comment by JackiePease** [comment]
It still isn't working - everyone's using the outside one

**Comment by MatthewCroughan** [comment]
That's right. I've been aware of this since #1417 was created, as Adrian told me over the phone after I woke him up to let me in, I think I saw this mentioned somewhere else, though I cannot remember if it was in person, vocally, etc.

**Comment by johnmckerrell** [comment]
Ok, well, to confirm, it's definitely online. Looks like it has an uptime a few days shorter than the last scan so I'm going to guess someone already tried rebooting so that would suggest it /is/ a physical problem. Has anyone done any basic checking that the RFID hasn't fallen back? (Not pushing for action just collecting information)

**Comment by MatthewCroughan** [comment]
That would probably not be the case, as putting the fob up to it makes the red LED light up, as can be seen behind the translucent dots on the enclosure. @johnmckerrell

**Comment by MatthewCroughan** [comment]
Although, I failed to mention in #1417 that this still occurred for that doorbot too. It was simply that one of the duponts had come loose, the one responsible for data/signalling in one direction. So it's possible that the communication with the door has simply failed due to one wire. 

Should I check when I'm next in?

**Comment by johnmckerrell** [comment]
Ah, yeah suggests a data wire has become detached. In theory that could happen due to a system/software issue too but everything looks fine, and I've just rebooted it to make sure. Yes if you don't mind checking next time you're in that would be good.

**Comment by amcewen** [comment]
I had a look a day or two after it was first reported but didn't get to updating here (annoyingly). `logcards` is running on it fine, and a reboot didn't help at all.  

I've just pulled the front off it (I think I did that the other day too, but couldn't remember for definite) and the wires look okay - the only wire that isn't connected is the `input` pin that we don't use.  The next thing I was going to try was swapping in a spare reader to see if that's troublesome - I think I've got a spare or two left over from when Museum in a Box used that reader.

**Comment by johnmckerrell** [comment]
I'm guessing you can do that swap without soldering so that would be good to try. I had an idea that we'd had issues at the Pi end before which I've found to be #905. Obviously we've managed a good while since then and there really shouldn't be any reason that the Pi would be disturbed but if swapping the reader doesn't help then that would be the next step. You do need very long ladders to get up to them, I don't remember if I've managed it from the ones DoES has but if I have it wouldn't have been very safe!

**Comment by amcewen** [comment]
I went to try swapping the reader but found that the reader is fine.  Re-seating the (seemingly fine already) dupont connector brought it back to life.  I'm going to close this for now, we can revisit it if (a) the problem recurs or (b) we rework the wiring in #1417 and can apply the same fix here.

**Comment by johnmckerrell** [comment]
This occurred again on Thursday. I had a go with it, thought it was that the reader was too far back so I actually hot glued it to the front. It would work until I closed up the box. I think the dupont needs resoldering, or possibly just re-seating as I didn't actually try that. I'm afraid I popped a blob of glue on the connector hoping that would keep it in place but obviously not, so hopefully that won't get in the way too much if re-soldering (or even re-seating).
